subprojects/kotlin-dsl-plugins/src/main/kotlin/org/gradle/kotlin/dsl/plugins/dsl/KotlinDslCompilerPlugins.kt

```diff
@@ -60,6 +71,10 @@ abstract class KotlinDslCompilerPlugins : Plugin<Project> {
                     it.setWarningRewriter(ExperimentalCompilerWarningSilencer(listOf(
                         "-XXLanguage:+DisableCompatibilityModeForNewInference",
                         "-XXLanguage:-TypeEnhancementImprovementsInStrictMode",
+                        // Kotlin 1.8.0 has wrong warning message for assign plugin,
+                        // replace with 'Assign Kotlin compiler plugin is an experimental feature' once we update to 1.8.20.
+                        // https://github.com/JetBrains/kotlin/commit/0eb34983cb38064684cfc76dacb8d4460fcc6573
+                        "Lombok Kotlin compiler plugin is an experimental feature",
```

Comment:
:grey_question: We could consider adding both messages already.
When 1.8.20 will be out, early adopters will use it before we get a change to upgrade.

Reply:
I planned to add other extension methods in follow-up PR, but I can do that know.

Reply:
More assignment extensions could go into another PR, me thinks. It was just me being curious and wanting to make sure we don't forget.

Reply:
+1 to keeping this small and instead creating issues to make sure we don't forget adding more stuff.

Reply:
@lptr are you referring to my original comment in this thread about the compiler warning or to the other extension methods subject?

Reply:
I meant the extension methods.

Reply:
Added additional message, also I added incubating message. Message is triggered from `assign` methods.